src/commands/help.js

```diff
@@ -0,0 +1,35 @@
+const Discord = require('discord.js')
+module.exports = {
+  prefix: '!help',
+  /**
+   * @name help
+   * Displays currently available commands.
+   *
+   * @param {Discord.Message} message the message provided
+   */
+
+  /**
+   *Use this format to add new commands:
+   *{name: "Command Name", value: "Command Description"},
+   */
+
+  command: function help(message) {
+    console.log('help', message);
+      const helpEmbed = new Discord.MessageEmbed()
+      .setColor('#0099ff')
+      .setTitle('Bot Information')
+      .setDescription("Hello! I am a test bot created by brantaniguchi and members of FreeCodeCamp.org to experiment with the process of building a Discord Bot. You can view my source code at https://github.com/bradtaniguchi/discord-bot-test")
+      .addFields(
+        { name: '\u200B', value: '\u200B' },
+        {name: "Commands", value: "Here are my currently available commands!"},
+        {name: "!eightball <optional string>", value: "Returns a response from a magic 8 ball toy"},
```

Comment:
The magic 8 ball is not a toy! Its a magic pool ball!

Reply:
FIXED